doc/legacy/dml_pseudocode.rst.txt

```diff
@@ -0,0 +1,150 @@
+API and Pseudocode for General Double ML Estimator

```

Comment:
Why create this file?  We can always look at the history in git if we need to retrieve it.

Reply:
agreed

Reply:
removed it